micromagneticdata/tests/test_drive.py

```diff
@@ -80,3 +82,24 @@ def test_ovf2vtk(self):
     def test_slider(self):
         for drive in self.data:
             assert isinstance(drive.slider(), ipywidgets.IntSlider)
+
+    def test_to_xarray(self):
+        for drive in self.data:
```

Comment:
We should add a MinDrive with mutiple steps (if we don't already have one). But maybe better in a separate PR?